doc/services/storage/zms/zms.rst

```diff
@@ -413,52 +394,48 @@ and this section provides information to help make a choice.
 More generally to make the right choice between NVS and ZMS, all the blockers should be first
 verified to make sure that the application could work with one subsystem or the other, then if
 both solutions could be implemented, the best choice should be based on the calculations of the
-life expectancy of the device described in this section: :ref:`wear-leveling`.
+life expectancy of the device described in this section: `Wear leveling <#wear-leveling>`_.
 
 Recommendations to increase performance
 ***************************************
 
 Sector size and count
 =====================
 
-- The total size of the storage partition should be well dimensioned to achieve the best
-  performance for ZMS.
+- The total size of the storage partition should be set appropriately to achieve the best
+  performance with ZMS.
   All the information regarding the effectively available free space in ZMS can be found
-  in the documentation. See :ref:`free-space`.
-  We recommend choosing a storage partition that can hold double the size of the key-value pairs
+  in the documentation. See `Available space for user data <#Available-space-for-user-data-key-value-pairs>`_.
+  It's recommended to choose a storage partition size that is double the size of the key-value pairs
   that will be written in the storage.
-- The size of a sector needs to be dimensioned to hold the maximum data length that will be stored.
-  Increasing the size of a sector will slow down the garbage collection operation which will
-  occur less frequently.
-  Decreasing its size, in the opposite, will make the garbage collection operation faster
-  which will occur more frequently.
+- The sector size needs to be set to be able to hold the maximum data size that will be stored.
+  Increasing the sector size will slow down the garbage collection operation and make it occur
+  less frequently.
+  Decreasing its size, on the opposite, will make the garbage collection operation faster but also
+  occur more frequently.
 - For some subsystems like :ref:`Settings <settings_api>`, all path-value pairs are split into two ZMS entries (ATEs).
-  The header needed by the two entries should be accounted when computing the needed storage space.
-- Using small data to store in the ZMS entries can increase the performance, as this data is
+  The headers needed by the two entries should be accounted for when computing the needed storage
+  space.
+- Storing small data (<= 8 bytes) in ZMS entries can increase the performance, as this data is
   written within the entry header.
```

Comment:
Entry or entry header?

Reply:
entry header is confusing as there are some special entries called header entries.
Will change that to "entry"